lib/core/Canvas.js

```diff
@@ -395,7 +395,11 @@ Canvas.prototype.setActivePlane = function(plane) {
   }
 
   if (typeof plane === 'string') {
-    plane = this.getPlane(plane);
+    var name = plane;
+    plane = this.getPlane(name);
+    if (!plane) {
+      throw new Error('plane with name ' + name + ' does not exist');
```

Comment:
We do adopt the `<abc> does not exist` notation in many places. Does it make sense to do that here, too?

Reply:
I think it does make sense to be consistent. Would it make sense to also refactor it here https://github.com/bpmn-io/diagram-js/blob/f55993279282b5b0393fc7c4638b9a4d800e005c/lib/core/Canvas.js#L687

to `<element.id> already exists` as well?